### DISCUSSION TITLE
Is auth.js GDPR compliant?

### DISCUSSION THREAD

**Discussion by clibequilibrium** [nextauthjs/next-auth#9522] [Help]
Hi !

If I process data inside EU and use one of the database adapters, is it fair to say that auth.js is GDPR compliant? Since there is no username that is stored and persistent in the database and only userId. 

Thank you

**Comment by alpap** [comment]
I don't believe so as it seems to store the name and email unencrypted

```js
export const users = pgTable("user", {
 id: text("id").notNull().primaryKey(),
 name: text("name"),
 email: text("email").notNull(),
 emailVerified: timestamp("emailVerified", { mode: "date" }),
 image: text("image"),
})
```
